include/OpenColorIO/OpenColorIO.h

```diff
@@ -755,9 +755,10 @@ class OCIOEXPORT Config
                                             const char * dstName,
                                             const char * dstInterchangeName);
 
+    ~Config();
```

Comment:
Could you add a friend declaration instead of making the destructor public?

Reply:
Possibly, but I'm not sure who the friend would be declared as, without creating a pybind11 dependency in OCIO core.

Reply:
pybind11 does offer a way to handle a private destructor with `unique_ptr`, but didn't work with `shared_ptr` as the holder type.

Reply:
Right.